kani-compiler/src/codegen_cprover_gotoc/codegen/intrinsic.rs

```diff
@@ -120,16 +120,71 @@ impl<'tcx> GotocCtx<'tcx> {
         }
     }
 
-    /// c.f. `rustc_codegen_llvm::intrinsic` `impl IntrinsicCallMethods<'tcx> for Builder<'a, 'll, 'tcx>`
-    /// `fn codegen_intrinsic_call`
-    /// c.f. <https://doc.rust-lang.org/std/intrinsics/index.html>
+    fn codegen_ctpop(
+        &mut self,
```

Comment:
Nit: can you please move these aux functions to be after the codegen_intrinsics?